test/api_addProductImage_test.dart

```diff
@@ -1,3 +1,5 @@
+import 'dart:math';
```

Comment:
In general, we import the "math" library with a prefix: `import 'dart:math' as math;`
and all related calls will now look like this: `math.xxx`